tools/image_processing/imagej2/imagej2_bunwarpj_raw_transform/imagej2_bunwarpj_raw_transform.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_bunwarpj_raw_transform" name="Apply raw transformation" version="1.0.0">
+    <description>with bUnwarpJ</description>
+    <macros>
+        <import>imagej2_macros.xml</import>
+    </macros>
+    <expand macro="fiji_headless_requirements" />
+    <command>
+<![CDATA[
+    python $__tool_directory__/imagej2_bunwarpj_raw_transform.py
+    --target_image "$target_image"
+    --target_image_format $target_image.ext
+    --source_image "$source_image"
+    --source_image_format $source_image.ext
+    --raw_transformation $raw_transformation
+    --source_out "$source_out"
+    --source_out_datatype $source_out_datatype
+    --jython_script $__tool_directory__/jython_script.py
+    --max_heap_size_type $set_max_heap_size.max_heap_size_type
+    --max_heap_size $set_max_heap_size.max_heap_size
+]]>
+    </command>
+    <inputs>
+        <param format="bmp,eps,gif,html,jpg,pcx,pgm,png,psd,tiff" name="target_image" type="data" label="Target image"/>
+        <param format="bmp,eps,gif,html,jpg,pcx,pgm,png,psd,tiff" name="source_image" type="data" label="Source image"/>
```

Comment:
@bgruening Oops!  Probably not.  I'll eliminate HTML options for all of the ImageJ tools as part of my next commit.  Thanks for catching!